Pushover.indigoPlugin/Contents/Server Plugin/plugin.py

```diff
@@ -109,6 +114,11 @@ def send(self, pluginAction):
 
 		self.debugLog(u"Result: %s" % r.text)
 
+		if r.status_code == 200:
+			self.logger.info(u"Pushover announceemnt was sent sucessfully, title: " + msgTitle + ", body: " + msgBody)
```

Comment:
"announcement" is spelled wrong, but should probably be "message" or "notification" anyway

Reply:
That's already been fixed.  But 'message' would probably be better.

Reply:
fixed (again) in 1.5.1